CHANGELOG.md

```diff
@@ -37,7 +37,7 @@
  * Add translatable contact email placeholder #259
  * Rename 'team name' to 'team identifier' #258
  * Add translatable footer links #253
- * Show motivationless attributes on entity detail #255
+ * Show motivationless attributes on entity detaOil #255
```

Comment:
Is this an intended change?

Reply:
Reverted that later, I also added an empty `__callStatic` method somewhere

https://giant.gfycat.com/IdealCooperativeGnat.webm